src/data-table/tests/DataTable.spec.tsx

```diff
@@ -8,6 +8,38 @@ describe('n-data-table', () => {
   it('should work with import on demand', () => {
     mount(NDataTable)
   })
+  it('show custom empty', () => {
+    const columns = [
+      {
+        title: 'Name',
+        key: 'name'
+      },
+      {
+        title: 'Age',
+        key: 'age'
+      },
+      {
+        title: 'Address',
+        key: 'address'
+      },
+      {
+        title: 'Tags',
+        key: 'tags'
+      },
+      {
+        title: 'Action',
+        key: 'actions'
+      }
+    ]
+    const data: any = []
+    mount(() => (
```

Comment:
You need to assert something. For example `expect(wrapper.find(...).exists()).toEqual(true)`